pandas/core/frame.py

```diff
@@ -1363,7 +1363,14 @@ def to_html(self, buf=None, columns=None, col_space=None, colSpace=None,
         max_cols : int, optional
             Maximum number of columns to show before truncating. If None, show
             all.
-
+        formatters : callabable or dict/list/tuple of callables, optional
+            The keys to the dict are the column names, any missing or additional
+            names will be given defaults or ignored respectively. If list/tuple
+            the length should match the columns exactly.
+            Each callable can have an optional boolean `escape` attribute, 
+            and an optional string `justify` attribute. See `_make_fixed_width` 
```

Comment:
Could you please give a brief description instead of referring to the source code? Also -- maybe add an example to the HTML a export docs? http://pandas-docs.github.io/pandas-docs-travis/io.html#io-html